random-reviewer-picker/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "Random reviewer picker"
+description: "Picks reviewers and maintainers for PRs reviews from a predefined list"
+author: "Emmanuel Viala"
+inputs:
+  GITHUB_TOKEN:
```

Comment:
Can we make casing of this input as consistent as other input's as well ?

Reply:
The rule there was that secrets are uppercase and variables lowercase

Reply:
Okay, new rule learnt today :)